firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## Version 0.1.4
+
+fixed - Converting circular structure to JSON error (fixes issue #236).
+fixed - Fixed bug where modules were not sharing the same Firestore
+DocumentReference. (fixes issue #265)
+fixed - Updated @firebaseextensions/firestore-bigquery-change-tracker dependency (fixes issues #250 and #196).
```

Comment:
```suggestion
fixed - Updated @firebaseextensions/firestore-bigquery-change-tracker dependency. (Issues #250 and #196)
```

Reply:
I wonder if this is a "changed" or a "fixed" update?

Reply:
Hmm - I'd leave it as fixed, because the fundamental behavior of the extension hasn't changed